modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -1404,6 +1407,93 @@ public String getName() {
         return visible;
     }
 
+    /**
+     * Indicates whether or not this {@code Node} is shown. A node is considered shown if it's
+     * part of a {@code Scene} that is part of a {@code Window} whose
+     * {@link Window#showingProperty showing property} is {@code true}. The {@link Node#visibleProperty visibility}
```

Comment:
I think you should choose one of the following:
`...whose {@link Window#showingProperty} is...`
`...whose {@link Window#showingProperty showing} property is...`

Reply:
Updated to second variant